reflex/.templates/apps/sidebar/code/styles.py

```diff
@@ -5,10 +5,10 @@
 border_radius = "0.375rem"
 box_shadow = "0px 0px 0px 1px rgba(84, 82, 95, 0.14)"
 border = "1px solid #F4F3F6"
-text_color = "black"
-accent_text_color = "#1A1060"
-accent_color = "#F5EFFE"
-hover_accent_color = {"_hover": {"color": accent_color}}
+text_color = "var(--text-1)"
+accent_text_color = "var(--accent-10)"
```

Comment:
I think you can do `rx.color("accent", 10)`

Reply:
you can _now_, that was literally added days ago 😁 #2713